book/website/community-handbook/local-build.md

```diff
@@ -1,154 +1,177 @@
+The next steps use {term}`Makefile`.
+The Makefile contains instructions to build a set of "targets".
+That way we can easily run the same commands repeatedly, and in different environments, without needing to remember all the parameters.
+It is easiest to change into the directory containing the Makefile,
 
-9. And now build the book:
+```
+$ cd book
+```
 
-   ```console
-   jupyter-book build .
-   ```
+Install the build dependencies into your virtual environment,
 
-10. The output of the build process will provide output such as below that demonstrate how you can view the book locally:
+```console
+$ make deps
+```
 
-    ```text
-    ===============================================================================
+### Build the Book
 
-    Finished generating HTML for book.
-    Your book's HTML pages are here:
-        _build/html/
-    You can look at your book by opening this file in a browser:
-        _build/html/index.html
-    Or paste this line directly into your browser bar:
-        file:///[...]/the-turing-way/the-turing-way/book/website/_build/html/index.html
+You are now ready to build the book.
+You can build the book with,
 
-    ===============================================================================
-    ```
+```console
+$ make book
+```
 
-### Build the book while working on a Pull Request
+The output of the build process will provide output such as below that demonstrate how you can view the book locally,
 
-If you would like to preview a version of the book from a certain branch (perhaps to render the book while working on a PR) then simply switch to the required branch and rebuild the book as in step 9:
+```text
+===============================================================================
 
-   ```bash
-   git checkout mybranch
-   jupyter-book build .
-   ```
+Finished generating HTML for book.
+Your book's HTML pages are here:
+    _build/html/
+You can look at your book by opening this file in a browser:
+    _build/html/index.html
+Or paste this line directly into your browser bar:
+    file:///<path to repository>/book/website/_build/html/index.html
 
-Follow the link as before and you will see changes specific to that branch rendered.
+===============================================================================
+```
 
-### Clean up a recent build
+Open `index.html` in your web browser to look at your local build.
 
-When you test your edits by building the book multiple times, it is better to clean up the last build before generating a new one.
-You can either manually delete the `book/website/_build` folder every time, or run this command:
+### Other Targets
+
+There are other build targets you can use.
+The `strict` target is useful for debugging.
+It will make any warnings raise an error, but also continue the build.
+That way, all warnings should be presented to you as errors.
+Run the strict build with,
+
+```console
+$ make strict
+```
+
+You can build the book with user pathways with,
```

Comment:
Might be helpful to have something of link to something that explains what user pathways are.

Also it's a little unclear if this replaces the build process without pathways or adds the pathways to an existing build.

Reply:
Yeah, I did think about that but I don't think there is currently and explanation of the pathways. It felt out of scope for this PR.

What actually happens is, some extra markdown files are generated and a build is run. I'm not sure how much the details matter, because either way you end up with a book including the pathways. Is there are wording you think would be better?

Reply:
Reasonable that the pathway explanation is out of scope.

Re-reading it the sentence 'there are other build targets you can use' makes it a little clearer. But I think concept of a build target may need a tiny bit of extra context. Maybe a clarifying sentence that these are *alternate* ways of building the whole book, not for example extra optional build steps that you might run after `build book`.

I think reading this I'd expect `make pathways` to be necessary to get pathways in the build. With the information that it's not, I'd be a little unclear on the reason for the `make pathways` build to exist, if you get essentially the same thing as `make book`? Is it just extra debug info for pathways like `make strict`? As a user I don't think I'd have enough info to pick which of these I'd want to use and why so would just be confused by the choice.

Reply:
Hmm, yes. The real truth is

```Makefile
book:
	jupyter-book build ./website

strict:
	jupyter-book build ./website -W --keep-going

pathways:
	pathways --build ./website/
```

So `book` and `pathways` are different. From a _clean_ state, `book` would build the book without pathways, `pathways` would build the book with pathways. I think you're right that a bit more text will make that clearer.

Reply:
Updated in 6c54700